homeassistant/components/hyperion/__init__.py

```diff
@@ -5,6 +5,7 @@
 from typing import Any, Optional, Tuple
 
 from hyperion import client, const as hyperion_const
+from semver import VersionInfo
```

Comment:
We can use the tool in `pkg_resources` that comes with `setuptools`.
```
Python 3.8.5 (default, Jul 28 2020, 12:59:40) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.18.1 -- An enhanced Interactive Python. Type '?' for help.

In [1]: from pkg_resources import packaging

In [2]: packaging.version.parse("2.0.0-beta") > packaging.version.parse("2.0.0-alpha.8")
Out[2]: True

In [3]: packaging.version.parse("2.0.0-alpha.7") > packaging.version.parse("2.0.0-alpha.8")
Out[3]: False
```

Reply:
Nice!